lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -280,6 +282,13 @@ def track_lines(
     )
 
 
+@deprecated(
```

Comment:
The parameter list will throw a warning on the docs build with `napoleon`. Should probably axe it.